src/status_im/chat/subs.cljs

```diff
@@ -356,3 +359,10 @@
   :<- [:get-active-chats]
   (fn [chats _]
     (apply + (map #(count (:unviewed-messages %)) (vals chats)))))
+
+(reg-sub
+  :message-contains-only-emoji
+  (fn [_ [_ text]]
+    (let [length-threshold 20]
```

Comment:
So the limit is 20, not 10?

Reply:
You should not abuse hotdog! :D

Reply:
👍

Reply:
if this is the case it might be worth mentioning in a comment 🙂